test/test-tarantool/iproto.test.py

```diff
@@ -0,0 +1,55 @@
+"""
+Original Tarantool's test box-py/iproto.test.py had a problem when output with
+running under Python 2 was not the same as with running under Python 3.
+
+Fixed in commit 697b79781cc63e2d87d86d43713998261d602334
+"test: make output of box-py/iproto.test.py deterministic".
```

Comment:
Is this test requires a special handling on the test-run side? Or here we just verify the `Connection` class from the tarantool-python connector provided by test-run?

Similarly to the [comment](https://github.com/tarantool/test-run/pull/283#discussion_r603731146) regarding `call.test.py`: you link a problem around Python 3 compatibility *of the test itself*. It is interesting, but it does not reveal motivation of your choice to add it here.

Reply:
You are right, nothing special for test-run itself in a test. To support Python 3 in call.test.py there were commits to tarantool-python and test itself. But I would keep test as integration check to make sure call.test.py is not broken. Althugh probably it is better to add a unit test to tarantool-python to cover this case.

Reply:
Okay.